### ISSUE TITLE
Set budget amount

### ISSUE THREAD

**Issue by ianbro** [ianbro/BudgetTracker#4]
As a Budget Planner, I should be able to set the amount available in a budget so that I can plan how much I want to be able to spend in that budget.

The budget can take on a dollar amount or a percentage amount. If the percentage amount is used, then this budget will have a calculated amount that is that percent of it's parent budget's dollar amount. Of course, this assumes that the parent budget has a set dollar amount. If not, then this budget can't have a percentage amount.

**Comment by BenjaminGroseclose** [comment]
@ianbro Believe this is complete by UpdateBudget

**Comment by ianmann56** [comment]
Yeah. I had left it open for the UI side but that's in a different repo. However, there are more tasks that deal with updating budget amounts so this should stay open. I'll link them.
